dask_cugraph/tests/test_karate_csv.py

```diff
@@ -38,15 +40,16 @@ def test_pagerank():
     import dask_cugraph.pagerank as dcg
     input_df = cudf.DataFrame()
     print("Read Input Data.")
-    input_data_path = r"datasets/karate.csv"
     ddf = dcg.read_csv(input_data_path, delimiter=' ', names=['src', 'dst', 'value'], dtype=['int32', 'int32', 'float32'])
     print("CALLING DASK MG PAGERANK")
     pr = dcg.mg_pagerank(ddf)
-    print(pr)
     res_df = pr.compute()
-    #res_df = f_pr.sort_values('pagerank')
-    #print(len(res_df))
-    for c in range(len(res_df)):
-        print(res_df['vertex'][c], res_df['pagerank'][c])
+    err = 0
+    tol = 1.0e-02
```

Comment:
I think decreasing `tol` to `1.0e-05` and adding a number of iteration of _at least_ `50` to  `dcg.mg_pagerank` would make the test stronger.

Reply:
the max iter is 10 currently in cugraph wrapper, max iter is not included in the api for dask_cugraph call.

Reply:
the api needs to include max iter and alpha too

Reply:
let's expose this in the API then

Reply:
will be addressed in #30